filecoin-proofs/src/api.rs

```diff
@@ -762,38 +762,40 @@ mod tests {
     #[test]
     #[ignore]
     fn test_pip_lifecycle() -> Result<(), failure::Error> {
-        let number_of_bytes_in_piece: u64 = 500;
-        let unpadded_number_of_bytes_in_piece = UnpaddedBytesAmount(number_of_bytes_in_piece);
-        let bytes: Vec<u8> = (0..number_of_bytes_in_piece)
+        // This size is a minimal repro for the P0 API sanity check failure.
+        let sector_size = 1 << 14;
```

Comment:
@porcuquine 

Using a sector of `1 << 14` size here is going to trigger the lazy Groth parameter generation flow when the `test_ignored_release` CircleCI build runs. Longer builds and all that.

Do you mind if I change `let sector_size = 1 << 14;` to `let sector_size = TEST_SECTOR_SIZE;`? I realize that the old bug wouldn't manifest with 1024B sectors and I'm not concerned about that.